src/redux/missions/missions.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-unused-vars */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const fetchMission = createAsyncThunk('mission/fetchMission',
+  async (id = null, { rejectWithValue }) => {
+    try {
+      const response = await axios.get('https://api.spacexdata.com/v3/missions');
+      return response?.data;
+    } catch (error) {
+      return rejectWithValue(error.response.data);
+    }
+  });
+const missionSlice = createSlice({
+  name: 'missions',
+  initialState: {
+    missionList: [],
+    status: null,
+    error: null,
+  },
```

Comment:
- [ ] You are required to filter the results from the API after fetching the missions, before dispatching to the store. Right now only the rockets values are filtered. Kindly filter the missions as well and save ONLY these values in the store: `mission_id`, `mission_name`, `description`.

  ![irr](https://user-images.githubusercontent.com/54848363/204012718-7522e234-5480-4326-8000-9eaf98329daf.jpg)

Reply:
Noted with thanks